ocaml/xapi/test_vdi_allowed_operations.ml

```diff
@@ -50,52 +50,60 @@ let run_assert_equal_with_vdi ~__context ?(cmp = my_cmp) ?(ha_enabled=false) ?sm
    code. This DO NOT fully test the aforementionned function *)
 let test_ca98944 () =
   let __context = Mock.make_context_with_new_db "Mock context" in
+
+  (* Create a non-snapshot VM that the VDI will be attached to *)
```

Comment:
What would happen if instead we just changed the definition of `Test_common.make_vbd` to use `make_vm ~__context ()` for its default for its `vM` parameter, instead of its current `?(vM=Ref.make ())`?

Reply:
I'd prefer to leave it as it is, to stick to the conventions of `Test_common` - these are very simple functions wrapping the Db functions with some sensible defaults. Also, I do not know what effect it would have on existing tests. So it's best not to change it, because existing tests may rely on the default value.